query-engine/prisma/src/tests/type_mappings/mysql_types.rs

```diff
@@ -181,11 +182,12 @@ async fn mysql_types_roundtrip(api: &TestApi) -> TestResult {
                     string_text_text
                     string_text_mediumtext
                     string_text_longtext
-                    string_binary_binary
-                    string_blob_tinyblob
-                    string_blob_mediumblob
-                    string_blob_blob
-                    string_blob_longblob
+                    # string_binary_binary
+                    # string_binary_varbinary
+                    # string_blob_tinyblob
+                    # string_blob_mediumblob
+                    # string_blob_blob
+                    # string_blob_longblob
```

Comment:
These were mapped to strings and worked, but it's not correct, so also commented out.